lib/paramedic.js

```diff
@@ -94,8 +96,8 @@ ParamedicRunner.prototype.run = function () {
             self.killEmulatorProcess();
         }
         self.cleanUpProject();
-        return result;
-    });
+    })
+    .fin(self.displaySauceDetails.bind(self));
```

Comment:
Is it possible to call this method from `fin` handler above? I don't see the reason for attaching `fin` to promise chain twice